### PR TITLE
Change to use new private syntax and other minor improvements

### PR DESCRIPTION
**Pull request by nikolauska**
This merge request mainly change private to new syntax where private is added with variable and not separately.  There are also some whitespace fixes and other minor improvements.

### CODE REVIEW

**Comment by dedmen** [comment]
Why is using private _variable better than private ["_variable1","_variable2"] ?

File: addons/core/functions/fnc_isSpeaking.sqf [hunk 1]
```diff
@@ -1,5 +1,28 @@
 #include "script_component.hpp"
 
+<<<<<<< HEAD
```
Comment (dedmen) [inline addons/core/functions/fnc_isSpeaking.sqf]: nope!

Comment (nikolauska) [inline addons/core/functions/fnc_isSpeaking.sqf]: This has now been fixed, somehow managed to miss it

**Comment by nikolauska** [comment]
Having private before variable allows more easily to see scope it can be used in and where that variable has been initialized. Defining it as an array on top of the file means it can be used on whole file which is not really wanted behavior as you want to limit variable to only scope they are used.

Also problem that I saw in private ARRAY usage was that people tend to forget to update it when changing code where as having it on variable initialization makes it easier to remember.

Of course this it can look stupid in some of the files currently where there are dozens of variables defined, but I wanted to avoid changing code too much in this pull request and focus on just this.

**Comment by commy2** [comment]
`private keyword` is faster than `private command` (`private ARRAY` or `private STRING`).

If you compare older BI scripts like the Arsenal with newer ones like 3DEN, you can see that they use this newer syntax excusively now too.

**Comment by nikolauska** [comment]
I remembered there was talk about it, but couldn't find any info about it so I didn't mention it

File: addons/core/functions/fnc_generateSwSettings.sqf [hunk 2]
```diff
@@ -52,3 +52,4 @@ if (!_set) then {
 _sw_settings set [2, _sw_frequencies];
 
 _sw_settings
+>>>>>>> 8b8749952bfc75733054c34ae530f0cfdbf1d5bb
```
Comment (dedmen) [inline addons/core/functions/fnc_generateSwSettings.sqf]: möp! Also line 35 we are using 4 space tab. I'll fix that

**Comment by dedmen** [comment]
Do a playtest please. you can get the build from appveyor

**Comment by nikolauska** [comment]
There were two errors found on fnc_radioOn and fnc_onDDTangentReleasedHack on testing which have now been fixed.

File: addons/core/functions/fnc_onDDTangentReleasedHack.sqf [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 
-params ["_scancode"];
+params ["", "_scancode"];
```
Comment (dedmen) [inline addons/core/functions/fnc_onDDTangentReleasedHack.sqf]: could use param instead of params to only get the second parameter

**Comment by dedmen** [comment]
So. When you are sure you are done and everything is fine I will merge this. Better look over everything again to make sure it works fine. I did a new test build that you can test if needed.

File: addons/core/functions/fnc_currentDirection.sqf [hunk 4]
```diff
@@ -18,22 +18,19 @@
     Example:
         call TFAR_fnc_currentDirection
 */
-private ["_current_look_at", "_current_hyp_horizontal", "_current_rotation_horizontal"];
 
-_current_look_at = (screenToWorld [0.5,0.5]) vectorDiff (eyepos TFAR_currentUnit);
-_current_look_at params ["_current_look_at_x", "_current_look_at_y", "_current_look_at_z"];
+private _current_look_at = (screenToWorld [0.5,0.5]) vectorDiff (eyepos TFAR_currentUnit);
+private _current_look_at params ["_current_look_at_x", "_current_look_at_y", "_current_look_at_z"];
```
Comment (dedmen) [inline addons/core/functions/fnc_currentDirection.sqf]: whats this? I thought you tested this. This spammed my RPT in my test. 
-rep

Comment (nikolauska) [inline addons/core/functions/fnc_currentDirection.sqf]: I did test this but my RPT didn't show any issues, sorry for missing that :(

**Pull request merged**